Sources/KIF/Classes/KIFUIViewTestActor.m

```diff
@@ -385,6 +406,31 @@ - (void)swipeFromEdge:(UIRectEdge)edge
     [self.actor swipeFromEdge:edge];
 }
 
+- (void)activateCustomActionWithName:(NSString *)name;
+{
+    KIFUIObject *found = [[self usingCustomActionWithName:name] _predicateSearchWithRequiresMatch:YES mustBeTappable:NO];
+    
+    NSArray *actions = [found.element accessibilityCustomActions];
```

Comment:
NIT: Any reason to do this outside the block vs inside it? Don't feel strongly, just not what I would've expected. This is potentially moot if we refactor to using a helper method like `found.element customActionWithName:name` or something like that.

Reply:
mooted